src/sql/workbench/test/browser/modal/optionsDialogHelper.test.ts

```diff
@@ -10,9 +10,10 @@ import * as TypeMoq from 'typemoq';
 import * as assert from 'assert';
 import { ServiceOptionType } from 'sql/workbench/api/common/sqlExtHostTypes';
 import { $ } from 'vs/base/browser/dom';
+import { SelectOptionItemSQL } from 'sql/base/browser/ui/selectBox/selectBox';
```

Comment:
Would be good to add testing for value when value and text are different

Reply:
Done!

Reply:
Done!